### PR TITLE
javascript-forms - Simon Kim.

### PR DESCRIPTION
**Pull request by Skim-329**
jsformevents
jsformvalues

### CODE REVIEW

File: javascript-forms/form-events.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+var input = document.querySelector('input');
+var textarea = document.querySelector('textarea');
+
+input.addEventListener('focus', handleFocus);
+input.addEventListener('blur', handleBlur);
+input.addEventListener('input', handleInput);
+
+textarea.addEventListener('focus', handleFocus);
+textarea.addEventListener('blur', handleBlur);
+textarea.addEventListener('input', handleInput);
```
Comment (scbowler) [inline javascript-forms/form-events.js:24]: Looks good but you missed the `email` input

File: javascript-forms/form-values.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+function handleSubmit(event) {
+  event.preventDefault();
+  var formresults = {
+    name: contactform.elements.name.value,
+    email: contactform.elements.email.value,
+    message: contactform.elements.message.value
+  };
+  console.log(formresults);
+  document.querySelector('#contact-form').reset();
```
Comment (scbowler) [inline javascript-forms/form-values.js:11]: You have already selected the form, you don't need to re query for it. You can just use `contactForm.reset()`

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good but a couple updates needed. Let me know if you have any questions.

**Comment by Skim-329** [comment]
jsformevents
jsformvalues

**Review by scbowler — APPROVED** [review]
Looks good! 👍

**Pull request merged**